yt_dlp/cookies.py

```diff
@@ -454,7 +454,10 @@ def _extract_safari_cookies(profile, logger):
     cookies_path = os.path.expanduser('~/Library/Cookies/Cookies.binarycookies')
 
     if not os.path.isfile(cookies_path):
-        raise FileNotFoundError('could not find safari cookies database')
+        logger.info('Trying secondary cookie location')
```

Comment:
debug

Reply:
Should it be`logger.debug(...)` instead of `logger.info(...)`?

Reply:
@pukkandan, I hope this is correct now.